src/Symfony/Component/Notifier/Bridge/Telegram/README.md

```diff
@@ -76,6 +76,34 @@ $chatMessage->options($telegramOptions);
 $chatter->send($chatMessage);
 ```
 
+Adding Local Photo to a Message
```

Comment:
AFAICS the only change to the paragraph above is the relative file path, so I would rather extend this paragraph instead of copy pasting the whole one and just change a small line. WDYT?